orbs/shared/jobs/test.yaml

```diff
@@ -8,6 +8,7 @@ parameters:
     description: Username to use when fetching images from a registry
     type: string
   pre_setup_script:
+    default: ""
```

Comment:
`save_cache` also needs updating, I believe. I has this added as well.

Reply:
added it to the save_cache file